packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -369,6 +369,18 @@ const SidebarItemBase = forwardRef<any, SidebarItemProps>((props, ref) => {
   // Still waiting on a Q answered to fine tune the implementation
   const { isOpen } = useContext(SidebarContext);
 
+  const divStyle =
+    !isOpen && children ? { marginLeft: '24px', marginBottom: '4px' } : {};
```

Comment:
The arrow icon doesn't seem to be centralized horizontally. Was that intentional? 
<img width="1193" alt="Screenshot 2022-05-13 at 13 36 47" src="https://user-images.githubusercontent.com/25153114/168276975-45b62097-5c09-4124-9277-bbd4a95066ea.png">

If I look at the expanded version, it seems to be. And the class used for that `submenuArrow` adds a display flex to the element.

<img width="471" alt="Screenshot 2022-05-13 at 13 38 42" src="https://user-images.githubusercontent.com/25153114/168276978-c5600cb3-3fa1-4138-b70f-ef9b91c49806.png">

So maybe worth changing the styles to `{ display: 'flex', marginLeft: '24px' }` or re-use the submenuArrow class. What do you think? 
<img width="579" alt="Screenshot 2022-05-13 at 13 40 11" src="https://user-images.githubusercontent.com/25153114/168276980-02106a8e-c0b3-43d5-84aa-0987ad340490.png">